server/__test__/doesUserExist.test.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-undef */
+// eslint-disable-next-line no-undef
+import supertest from 'supertest';
+
+import { app } from '../app';
+import { buildSeed } from '../database/seed';
+
+beforeAll(() => buildSeed());
+
+describe('Testing doesEmailExist route', () => {
+  it('In the success case, the route should return 202, accepted status code.', (done) => {
+    supertest(app)
+      .get('/api/v1/user/check/sayed@gmail.com')
```

Comment:
u usually wanna do query `/api/v1/user/check?email=sayed@gmail.com`
instead of params right?

Reply:
Yes!!
It makes more sense to use than params for this use, thanks for that.